packages/installer/src/recipe-executor.tsx

```diff
@@ -31,11 +33,12 @@ export class RecipeExecutor<Options extends RecipeMeta> {
       await waitUntilExit()
     } catch (e) {
       log.error(e)
+      hasExecuted=false
       return
     }
-
-    log.info(
-      `\n🎉 The recipe for ${this.options.name} completed successfully! Its functionality is now fully configured in your Blitz app.\n`,
-    )
+    if(hasExecuted){
+      log.info(
+        `\n🎉 The recipe for ${this.options.name} completed successfully! Its functionality is now fully configured in your Blitz app.\n`,
+    )} 
```

Comment:
Hey @arjundubey-cr thank you!

This definitely solves the problem, but I think it'd be a bit cleaner to move the success log message up right after `await waitUntilExit()`. And then you don't need the new `hasExecuted` variable

Reply:
yes, it is a great idea indeed